epochStart/shardchain/validatorInfoProcessor.go

```diff
@@ -164,58 +163,70 @@ func (vip *ValidatorInfoProcessor) processAllPeerMiniBlocks(metaBlock *block.Met
 }
 
 func (vip *ValidatorInfoProcessor) computeMissingPeerBlocks(metaBlock *block.MetaBlock) {
-	missingNumber := uint32(0)
-	vip.mutMiniBlocksForBlock.Lock()
+	numMissingPeerMiniblocks := uint32(0)
+	vip.mutAllMissingPeerMiniblocks.Lock()
 
 	for _, mb := range metaBlock.MiniBlockHeaders {
 		if mb.Type != block.PeerBlock {
 			continue
 		}
 
-		vip.allPeerMiniblocks[string(mb.Hash)] = &miniBlockInfo{}
+		vip.mapAllMisingPeerMiniblocks[string(mb.Hash)] = nil
```

Comment:
the naming is confusing - this is exactly allPeerMiniblock - those which we found - not those which are missing

Reply:
Refactored